app/routes/questions/actions.tsx

```diff
@@ -39,10 +40,15 @@ const actions = {
     Icon: Followup,
     title: 'Followup',
   },
+  ping: {
+    Icon: Like,
```

Comment:
which icon should be used? There isn't a +1 icon, so I just used the thumbs up one for now

Reply:
OK, that's probably fine for now. I can create something different later if we decide that's needed. However, that icon should also to be selected/highlighted when the cursor is hovering over the questions, because clicking the question and icons performs the same function, correct?